### ISSUE TITLE
Angular: Step 2.5 Create todosList is missing root / of absolute path

### ISSUE THREAD

**Issue by kalmufti** [meteor/tutorials#77]
Need to add / before the imports dir or the example won't work and throw 404 error "HTML template does not exists"

```
export default angular.module('todosList', [
  angularMeteor
])
  .component('todosList', {
    templateUrl: '/imports/components/todosList/todosList.html',
      controller: TodosListCtrl
  });
```

**Comment by Cramertech** [comment]
Yes - please fix this code in the tutorial.  I spend about an hour quadruple checking my code everywhere because of this!! That's a frustrating way to get started with an introduction to Meteor.  Finally found this issue posted on github and fixed the problem I was having.  Come on guys - quality check your starter code. Boo.... :(

**Comment by lazaridis-com** [comment]
> Come on guys - quality check your starter code. Boo.... :(

@abernix, @hwillson - is this one confirmed?

Updating the tutorial is not trivial, as it fetches commits (this is not very maintainable).

The actual code is completely different:

https://github.com/meteor/simple-todos-angular
